config/routes.rb

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/216603539-2a4778b1-87db-4553-b501-a7f28c18cece.png)

- [ ] Kindly remove this section heading to make your readme look clean and professional :+1: